app/controllers/candidate_interface/prefill_application_form_controller.rb

```diff
@@ -50,23 +50,20 @@ def destroy_blank_application
     end
 
     def test_application_options
-      test_application_options = {
+      {
         recruitment_cycle_year: RecruitmentCycle.current_year,
-        states: [:unsubmitted_with_completed_references],
```

Comment:
need to check whether carry-over applications can be `:unsubmitted_with_completed_references`